lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -65,6 +65,7 @@
 #include "SPIRVUtil.h"
 #include "SPIRVValue.h"
 #include "VectorComputeUtil.h"
+#include "spirv/unified1/spirv.hpp"
```

Comment:
My editor inferred that and I overlooked it. Why it's default stance in the editor to do that?
There is a popular 'good practice' to avoid transitive headers inclusion. The principle is that if we directly using type/function/class from header than it should be included. The goal of this practice is to control how much headers are actually in use and in result decrease compile time.
I removed an inclusion.